mozjs/js/rust/magic_codegen/Cargo.toml

```diff
@@ -7,6 +7,7 @@ authors = ["zzhu <zzhu@mozilla.com>"]
 syn = "0.11"
 quote = "0.3"
 lazy_static = "0.2.1"
+log = "*"
```

Comment:
We shouldn't use `*` dependencies; use the latest version published on crates.io instead.